Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt

```diff
@@ -0,0 +1,44 @@
+In the previous videos of JSON Schema, we have shown the features and basic implementations of JSON Schema. In this video, we will deliver the actual validation process with libraries and examples.
+
+Throughout this video, we will mainly focus on validation with the JavaScript library AJV and the Python library          jsonschema, but there are many other libraries in different programming languages and tools to validate JSON Schema. 
```

Comment:
ajv pronounciation should be shorter, now it sounds like A  J  V. This happens in multiple places

Reply:
```suggestion
Throughout this video, we will mainly focus on validation with the JavaScript library AJV and the Python library          jsonschema, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema. 
```

Reply:
> ajv pronounciation should be shorter, now it sounds like A J V. This happens in multiple places

It is difficult to fix it though so we can leave as is.